src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -170,7 +169,7 @@ export default class PrimaryLayout extends Component {
       </Switch>
       <SocketListener location={location} />
       <SocketSubscriber type='community' id={get('slug', community)} />
-      <Intercom appID={isTest ? null : config.intercom.appId} hide_default_launcher={hasDetail} />
+      <Intercom appID={isTest ? null : config.intercom.appId} hide_default_launcher />
```

Comment:
this is just always hiding it right? so why not just remove?

Reply:
This is where it is invoked, as far as I can tell. I can move it somewhhere else?